src/pyhpp/coverage/path.cc

```diff
@@ -26,32 +26,32 @@
 
 #include <boost/python.hpp>
 #include <eigenpy/eigenpy.hpp>
-#include <hpp/coverage/path.hh>
-#include <hpp/coverage/cost.hh>
-#include <hpp/core/path.hh>
 #include <hpp/core/path-vector.hh>
+#include <hpp/core/path.hh>
 #include <hpp/core/roadmap.hh>
+#include <hpp/coverage/cost.hh>
+#include <hpp/coverage/path.hh>
 #include <hpp/pinocchio/device.hh>
 #include <pyhpp/util.hh>
 
 using namespace boost::python;
 
 namespace pyhpp {
 namespace coverage {
-using hpp::coverage::Path;
 using hpp::core::PathPtr_t;
 using hpp::core::PathVector;
 using hpp::core::PathVectorPtr_t;
+using hpp::coverage::Path;
 
 static PathPtr_t multiply(const PathPtr_t& p1, const PathPtr_t& p2) {
   Path ops;
   return ops.multiply(p1, p2);
 }
 
 static PathPtr_t createSpline(hpp::core::ConfigurationIn_t pose0,
-                               hpp::core::ConfigurationIn_t pose1,
-                               hpp::core::value_type length,
-                               hpp::core::size_type order) {
+                              hpp::core::ConfigurationIn_t pose1,
```

Comment:
please do not insert fake modifications. Let `pre-commit-ci` do the job. This will make merging of other modifications uselessly difficult.